### PR TITLE
Support from from_tag

### PR DESCRIPTION
**Pull request by phracek**
This pull request contains these fixes:

- [x] in Fedora land, `master` branch was renamed to `main` - 1st commit
- [x] from_tag added into configuration files - 2nd commit
- [x] fix some Python issues, like comments, variables - 3rd commit
- [x] Added functionality to support from_tag

### CODE REVIEW

**Comment by phracek** [comment]
@pkubatrh I would be glad for your review and comments.

File: container_workflow_tool/distgit.py [hunk 1]
```diff
@@ -100,40 +100,56 @@ def _bump_release(self, version_str, bump_type):
                 length[-1] = str(int(length[-1]) + 1)
             return '.'.join(length)
 
-    def _get_from(self, dockerfile_path):
+    def _get_tag(self):
+        return 'rhel' + self.base_image.split(':')[1].split('-')[0]
```
Comment (pkubatrh) [inline container_workflow_tool/distgit.py:104]: I do not think we need rhel in Fedora :)

Comment (phracek) [reply container_workflow_tool/distgit.py:104]: function `_get_tag` is not used at all ;)

**Comment by pkubatrh** [comment]
Otherwise lgtm

File: container_workflow_tool/distgit.py (hunk 1 above)
Comment (phracek) [reply container_workflow_tool/distgit.py:104]: Let's get rid of it by the next commit.

**Comment by phracek** [comment]
I have moved NodeJS container version from 15 -> 14.

Function `_get_tag` was removed because it is not used at all.

add git_path and git_branch into `cwt utils listupstream` command.

**Pull request merged**